docs/trex_release-notes.md

```diff
@@ -35,7 +35,7 @@ About this release:
 
   * `columnsToIncludeById` can be applied to `getSummaryDataAsync` to restrict the columns fetched.
 
-  * `includeDataValuesOnly` or `includeFormattedValuesOnly` can be applied to `getSummaryData` to restrict the amount of information returned to what you really need.
+  * `includeDataValuesOnly` or `includeFormattedValuesOnly` can be applied to `getSummaryDataAysnc` to restrict the amount of information returned to what you really need.
```

Comment:
Sorry... one more. It should by Async, rather than Aysnc.

Reply:
Right...!  fixing it now. Thanks.

Reply:
Fixed. Thanks for the corrections!